app/controllers/nonsipp/landorproperty/IndividualSellerNiController.scala

```diff
@@ -113,12 +113,12 @@ object IndividualSellerNiController {
   ): FormPageViewModel[ConditionalYesNoPageViewModel] =
     FormPageViewModel[ConditionalYesNoPageViewModel](
       "individualSellerNi.title",
-      Message("individualSellerNi.heading", individualName),
```

Comment:
Removal of `individualName` made this parameter redundant @MadStu .
As this branch is about to be merged, would it be possible to address this in the next branch to be merged?